superset-frontend/src/components/Radio/index.tsx

```diff
@@ -16,46 +16,60 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { styled } from '@superset-ui/core';
-import { Radio as AntdRadio } from 'antd';
+import { Radio as Antd5Radio } from 'antd-v5';
+import type {
+  RadioChangeEvent,
+  RadioProps,
+  RadioGroupProps,
+  CheckboxOptionType,
+} from 'antd-v5';
+import { Space, SpaceProps } from 'src/components/Space';
 
-const StyledRadio = styled(AntdRadio)`
-  .ant-radio-inner {
-    top: -1px;
-    left: 2px;
-    width: ${({ theme }) => theme.gridUnit * 4}px;
-    height: ${({ theme }) => theme.gridUnit * 4}px;
-    border-width: 2px;
-    border-color: ${({ theme }) => theme.colors.grayscale.light2};
-  }
+export type RadioGroupWrapperProps = RadioGroupProps & {
+  useSpace?: boolean;
+  direction?: SpaceProps['direction'];
+  spaceSize?: SpaceProps['size'];
```

Comment:
I think a better approach here could be something like this

`spaceConfig={...}`

If this is used then include the Space component. This is to avoid dependencies across different props.